src/components/composer/composer.ts

```diff
@@ -74,6 +74,8 @@ const TEMPLATE = xml/* xml */ `
     t-on-click.stop="onClick"
     t-on-blur="onBlur"
     t-on-paste.stop=""
+    t-on-compositionstart="onCompositionstart"
+    t-on-compositionend="onCompositionend"
```

Comment:
not sure what happens but those events are never fired on my machine.
Is it browser/os dependent?

Reply:
only fired when you use an IME, if you don't have one, you won't be able to test this

Reply:
See https://developer.mozilla.org/en-US/docs/Web/API/Element/compositionend_event  and https://developer.mozilla.org/en-US/docs/Web/API/InputEvent/isComposing

Reply:
Ah ok :+1: 
If it breaks something, it will break something already broken :laughing:

Reply:
taht's the idea  :p